content/blog/2022/02/2022-02-10-last-plugin-version-not-installable.adoc

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: "Why can't I install the latest plugin on my Jenkins LTS?"
+tags:
+- jenkins
+- plugin
+- upgrade
+- developer
+authors:
+- jmMeessen
+description: >
+  This blogpost explains why the latest version of a plugin can't be installed on Jenkins in some circumstances.
+---
```

Comment:
An opengraph image is used when the blog post is shared on social media (like twitter) and increases the reading rate .

This reuses the image from inside the blogpost.  If you prefer a different image, I've attached an alternative to this comment (from the [Jenkins Social Media Covers slide deck](https://docs.google.com/presentation/d/1Q1PgNnRTgzBpVRXPqQo3PudzCa2eoc6_1_NRjFRMLrU/edit?usp=sharing)).

```suggestion
  This blogpost explains why the latest version of a plugin can't be installed on Jenkins in some circumstances.
opengraph:
  image: /images/post-images/2022-02-10-last-plugin-version-not-installable/plugin-not-installable.png
---
```

![update-visible-but-cannot-install](https://user-images.githubusercontent.com/156685/153264815-6507e1f9-9715-40af-9af2-ed781e8360c5.png)